Python/Tests/Utilities.Python/PythonVisualStudioApp.cs

```diff
@@ -317,7 +317,7 @@ public PythonVisualStudioApp(IServiceProvider site)
 
         public ReplWindowProxy GetInteractiveWindow(string title, ReplWindowProxySettings settings = null) {
             var iwp = GetService<IComponentModel>(typeof(SComponentModel))?.GetService<InteractiveWindowProvider>();
-            var window = iwp?.AllOpenWindows.FirstOrDefault(w => ((ToolWindowPane)w).Caption == title);
```

Comment:
there's another one of these that you missed, in the same file

Reply:
thanks